fc/cli/task.py

```diff
@@ -31,7 +39,7 @@ def move(username: str, task_id: str, state: str):
         cli_library.fail_execution(1, 'Task search failed with {}'.format(exception))
 
     if the_task is not None:
-        the_task.transition(state)
+        the_task.transition(state, the_resolution, comment)
```

Comment:
Does JIRA allow setting the resolution or comment on a transition other than Resolved or Closed?  What happens?

Reply:
comments are ignored.  I believe adding a resolution when you don't need one results in a 400 error.